### ISSUE TITLE
Debuggability: Visualize dependants of a node

### ISSUE THREAD

**Issue by Janpot** [mui/mui-toolpad#523]
* Allow for tracing back the dependants of a certain UI element/query/... 
* In the same spirit we should warn before deleting an element that is being depended on
*

**Comment by prakhargupta1** [comment]
Does the View DOM option solve this?

![Image](https://user-images.githubusercontent.com/92228082/185236287-b395c815-17a3-4704-9fa9-53ae62976162.png)

**Comment by Janpot** [comment]
no, this issue is about warning when you delete something that something else still depends on. e.g. when you delete a connection that still has queries on a page. And possible also going further, showing the chain of dependencies across a page.
